docs/notebooks/hello_world.ipynb

```diff
@@ -157,6 +157,9 @@
    "diff_ignore": [
     "/metadata/language_info/version"
    ]
+  },
+  "nbsphinx": {
+   "orphan": true
```

Comment:
What is this for? Seems worth documenting since it might be important.

Reply:
I will be adding documentation to contributing about how to annotate drafts, internal-only, and orphaned pages. I have another PR for -draft and -internal annotations as well